[iconv] How to use with vcpkg? `vcpkg install iconv` then "Could not find a package configuration file"

Library name: iconv
CMake: https://cmake.org/cmake/help/latest/module/FindIconv.html


```cmake
include(FindIconv) # Attempted without this line also
find_package(iconv CONFIG REQUIRED)
```

After running this on 371be89c1e3bc42c71ee325193daa083e3d75d8f:
```
$ ./vcpkg install libiconv
Computing installation plan...
The following packages will be built and installed:
    libiconv[core]:x64-osx -> 1.16#11
Detecting compiler hash for triplet x64-osx...
Restored 1 packages from .cache/vcpkg/archives in 14.66 ms. Use --debug to see more details.
Starting package 1/1: libiconv:x64-osx
Installing package libiconv[core]:x64-osx...
Elapsed time for package libiconv:x64-osx: 2.29 ms

Total elapsed time: 2.654 s

The package libiconv provides CMake targets:

    find_package(iconv CONFIG REQUIRED)
    target_link_libraries(main PRIVATE Iconv::Charset)
```

NOTE: I am debugging this issue from Intl in vcpkg:
```
Undefined symbols for architecture x86_64:
  "_iconv", referenced from:
      __nl_find_msg in libintl.a(dcigettext.o)
  "_iconv_open", referenced from:
      __nl_find_msg in libintl.a(dcigettext.o)
  "_libiconv_set_relocation_prefix", referenced from:
      _libintl_set_relocation_prefix in libintl.a(relocatable.o)
```

Which made me think the wrong libiconv was being found, so I'm trying to explicitly specify the vcpkg one.

--- a/ports/libiconv/portfile.cmake
+++ b/ports/libiconv/portfile.cmake
@@ -26,15 +26,16 @@ if (NOT VCPKG_TARGET_IS_ANDROID)
     list(APPEND OPTIONS --enable-relocatable)
 endif()
 
-vcpkg_configure_make(SOURCE_PATH "${SOURCE_PATH}"
-                     DETERMINE_BUILD_TRIPLET
-                     USE_WRAPPERS
-                     OPTIONS
-                        --enable-extra-encodings
-                        --without-libiconv-prefix 
-                        --without-libintl-prefix
-                        ${OPTIONS}
-                    )
+vcpkg_configure_make(
+    SOURCE_PATH "${SOURCE_PATH}"
+    DETERMINE_BUILD_TRIPLET
+    USE_WRAPPERS
+    OPTIONS
+        --enable-extra-encodings
+        --without-libiconv-prefix
+        --without-libintl-prefix
+        ${OPTIONS}
+)
 vcpkg_install_make()
 
 vcpkg_copy_pdbs()
@@ -47,4 +48,6 @@ set(VCPKG_POLICY_ALLOW_RESTRICTED_HEADERS enabled)
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/share/${PORT}") # share contains unneeded doc files
 
+# Please keep, the default usage is broken
+file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/usage" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
 file(INSTALL "${SOURCE_PATH}/COPYING.LIB" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)

--- a/ports/libiconv/usage
+++ b/ports/libiconv/usage
@@ -1,0 +1,4 @@
+The package libiconv provides CMake targets:
+
+    find_package(Iconv)
+    target_link_libraries(main PRIVATE Iconv::Iconv)

--- a/ports/libiconv/vcpkg.json
+++ b/ports/libiconv/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "libiconv",
   "version": "1.16",
-  "port-version": 12,
+  "port-version": 13,
   "description": "GNU Unicode text conversion",
   "homepage": "https://www.gnu.org/software/libiconv/"
 }

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3626,7 +3626,7 @@
     },
     "libiconv": {
       "baseline": "1.16",
-      "port-version": 12
+      "port-version": 13
     },
     "libics": {
       "baseline": "1.6.5",

--- a/versions/l-/libiconv.json
+++ b/versions/l-/libiconv.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "cc4f1edf5a444934d4a757420a6cd3fed596e1d5",
+      "version": "1.16",
+      "port-version": 13
+    },
+    {
       "git-tree": "3942643a4d682982e61b60611394e4315dfc42a1",
       "version": "1.16",
       "port-version": 12

